JavaScript Hero - Object Mastery/script.js

```diff
@@ -0,0 +1,168 @@
+// Task 1
+// Object representing a superhero with properties like name, secretIdentity, powers, and weakness.
+const superhero = {
+  name: "Iron Hawk",
+  secretIdentity: "James Falcon",
+  powers: ["Flight", "Laser Vision", "Super Strength"],
+  weakness: "Kryptonite",
+};
+
+console.log(superhero); // Display superhero object
+
```

Comment:
Please remove all console logs for your projects that are not purely functional implementations.